projects/storefrontlib/cms-components/myaccount/my-account-v2-navigation/my-account-v2-navigation.component.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { MyAccountV2NavigationComponent } from './my-account-v2-navigation.component';
+import {
+  CmsComponentData,
+  NavigationNode,
+  NavigationService,
+} from '@spartacus/storefront';
+import { CmsNavigationComponent } from '@spartacus/core';
+import { of } from 'rxjs';
+import createSpy = jasmine.createSpy;
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'cx-navigation-ui',
+  template: '',
+})
+class MockNavigationUIComponent {
+  // @Input()
+  // dropdownMode = 'list';
```

Comment:
[MINOR]
Unused testing code can be removed

Reply:
done